components/places/src/ffi.rs

```diff
@@ -168,20 +170,25 @@ impl PlacesApi {
     }
 
     fn places_history_import_from_fennec(&self, db_path: String) -> Result<String> {
-        let metrics = import_history(self, db_path.as_str())?;
+        let metrics = import_fennec_history(self, db_path.as_str())?;
         Ok(serde_json::to_string(&metrics)?)
     }
 
     fn places_bookmarks_import_from_fennec(&self, db_path: String) -> Result<String> {
-        let metrics = import_bookmarks(self, db_path.as_str())?;
+        let metrics = import_fennec_bookmarks(self, db_path.as_str())?;
         Ok(serde_json::to_string(&metrics)?)
     }
 
     fn places_bookmarks_import_from_ios(&self, db_path: String) -> Result<()> {
-        import_bookmarks(self, db_path.as_str())?;
+        import_ios_bookmarks(self, db_path.as_str())?;
         Ok(())
     }
```

Comment:
tbh I think I'll kill the other migration functions and logic, didn't want to do it in the same PR, but maybe now is as good time as ever